app/src/main/java/com/handsonhip/model/Marketing.java

```diff
@@ -0,0 +1,91 @@
+package com.handsonhip.model;
+
+import jakarta.persistence.*;
+
+@Entity
+@Table(name = "marketing")
+public class Marketing {
+    //Table attributes
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    @Column(name = "marketingID")
+    private Long marketingID;
+
+    @ManyToOne
+    @JoinColumn(name = "productID", nullable = false)
+    private Product product;
+
+    @Column(name = "title")
+    private String title;
+
+    @Column(name = "description")
+    private String description;
+
+    @Column(name = "platforms")
+    private String platforms;
+
+    @Column(name = "suggestedPrice")
+    private Double suggestedPrice;
+
+    // No-arg constructor
+    public Marketing() {
+    }
+
+    // All-args constructor
+    public Marketing(Product product, String title, String description, String platforms, Double suggestedPrice) {
+        this.product = product;
+        this.title = title;
+        this.description = description;
+        this.platforms = platforms;
+        this.suggestedPrice = suggestedPrice;
+    }
+
+    // Getters and Setters
```

Comment:
We can use Lombok extension for creating Getters and Setters instead of creating each of them manually.